jwst/extract_2d/grisms.py

```diff
@@ -365,24 +376,31 @@ def extract_grism_objects(input_model,
 
     if grism_objects is None:
         # get the wavelengthrange reference file from the input_model
-        if ('wavelengthrange' not in reference_files or
-                reference_files['wavelengthrange'] in ['N/A', '']):
+        if "wavelengthrange" not in reference_files or reference_files["wavelengthrange"] in [
+            "N/A",
+            "",
+        ]:
             raise ValueError("Expected name of wavelengthrange reference file")
         else:
-            grism_objects = util.create_grism_bbox(input_model, reference_files,
-                                                   extract_orders=extract_orders,
-                                                   mmag_extract=mmag_extract,
-                                                   wfss_extract_half_height=wfss_extract_half_height,
-                                                   nbright=nbright)
-            log.info("Grism object list created from source catalog: {0:s}"
-                     .format(input_model.meta.source_catalog))
+            grism_objects = util.create_grism_bbox(
+                input_model,
+                reference_files,
+                extract_orders=extract_orders,
+                mmag_extract=mmag_extract,
+                wfss_extract_half_height=wfss_extract_half_height,
+                nbright=nbright,
+            )
+            log.info(
+                f"Grism object list created from source catalog: \
+                {input_model.meta.source_catalog}"
+            )
 
     if not isinstance(grism_objects, list):
         raise TypeError("Expected input grism objects to be a list")
     if len(grism_objects) == 0:
         raise ValueError("No grism objects created from source catalog")
 
-    log.info("Extracting %d grism objects", len(grism_objects))
+    log.info("Extracting {len(grism_objects)} grism objects")
```

Comment:
missing f-string

Reply:
Surprised ruff didn't catch that

Reply:
it can't tell if you mean the literal `{` or not